### PR TITLE
Reinitialize the whole thing when the window gets resized

### LINKED ISSUE #39: Screen height on orientation change
Would it be worth setting `screenY = window.innerHeight;` again within the window resize event listener? I'm thinking, when an ipad is rotated, the screen height changes.
Thanks

### PR DESCRIPTION
**Pull request by benface**
Fixes #39 and other issues I had.

### CODE REVIEW

File: rellax.js [hunk 1]
```diff
@@ -278,19 +271,27 @@
         var translate = 'translate3d(' + (self.options.horizontal ? positionX : '0') + 'px,' + (self.options.vertical ? positionY : '0') + 'px,' + zindex + 'px) ' + blocks[i].transform;
         self.elems[i].style[transformProp] = translate;
       }
-      self.options.callback(positions);
+      self.options.callback();
```
Comment (moeamaya) [inline rellax.js]: Curious why you removed the positions argument? I didn't write this code, but it seems like we pass that back as a return value on callback?

File: rellax.js [hunk 2]
```diff
@@ -101,34 +101,29 @@
...
+
+      blocks = [];
+
       screenY = window.innerHeight;
       screenX = window.innerWidth;
       setPosition();
 
       // Get and cache initial position of all elements
-      for (var i = 0; i < self.elems.length; i++){
+      for (i = 0; i < self.elems.length; i++){
```
Comment (moeamaya) [inline rellax.js]: Let's keep the var in there so it scopes the i value to this block

File: rellax.js (hunk 1 above)
Comment (benface) [reply rellax.js]: Oh yeah, sorry I didn’t mention that. It failed JSHint as `positions` is out of scope, and as I failed to notice the reference to that feature in the docs, I thought it was an error/leftover code. Still I’m not sure it works properly as `positions` is set in a loop?

File: rellax.js (hunk 2 above)
Comment (benface) [reply rellax.js]: I removed it because JSHint complained.

**Comment by benface** [comment]
Thanks for your review @moeamaya. I just pushed another commit to fix #88 and remove outdated comments. I also re-added the `var` I had removed in the `for` loop. Let me know what to do about passing `positions` to the callback.

**Comment by moeamaya** [comment]
Did you test a callback function and the positions still get passed it it?

**Comment by benface** [comment]
@moeamaya I just did and it does what is to be expected from that code: it passes the position (`x` and `y`) of the last block. That doesn't make much sense, does it? Should it pass an array of all the blocks' positions? I'm really not sure what it should do.

**Comment by moeamaya** [comment]
@susonwaiba Implemented this last year. Maybe he has some insights to that value?

**Comment by susonwaiba** [comment]
I don't know. I suppose it was for the callback. If it works, It works. 😄

**Comment by benface** [comment]
I don't see why anyone would rely on that unless they're sure to only ever have 1 rellax element in their page, but I don't really care and it's not the point of this PR, so I re-added the `positions` argument.

**Pull request merged**